### PR TITLE
fix max-series-per-database

### PR DESCRIPTION
**Pull request by wkloucek**

### CODE REVIEW

**Comment by fschade** [comment]
@wkloucek, do we really need to add a conf file. InfluxDB can be configured for most parts by using env variables. for example INFLUXDB_DATA_MAX_SERIES_PER_DATABASE.

File: config/influxdb/influxdb.conf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[meta]
+  dir = "/var/lib/influxdb/meta"
+
+[data]
+  dir = "/var/lib/influxdb/data"
+  engine = "tsm1"
+  wal-dir = "/var/lib/influxdb/wal"
+  max-series-per-database = 0
```
Comment (fschade) [inline config/influxdb/influxdb.conf:8]: is the conf really required?

File: docker-compose.yml [hunk 2]
```diff
@@ -48,6 +48,7 @@ services:
       - k6-net
     volumes:
       - ./config/influxdb/provisioning/users.sh:/docker-entrypoint-initdb.d/users.sh
+      - ./config/influxdb/influxdb.conf:/etc/influxdb/influxdb.conf
```
Comment (fschade) [inline docker-compose.yml:51]: is this required, we can use a env instead:

INFLUXDB_DATA_MAX_SERIES_PER_DATABASE=0

Comment (wkloucek) [reply docker-compose.yml]: I didn't find this on my first search. Now I changed the PR to this.

**Review by fschade — APPROVED** [review]
💯

**Pull request merged**